client/starwhale/api/_impl/data_store.py

```diff
@@ -123,22 +220,136 @@ def __str__(self) -> str:
         else:
             return self.name.upper()
 
-    __repr__ = __str__
+
+class SwCompositeType(SwType):
+    def __init__(self, name: str) -> None:
+        super().__init__(name, pa.binary())
+
+    def serialize(self, value: Any) -> Any:
```

Comment:
`def serialize(self, value: Any) -> Any:` -> `def serialize(self, value: Any) -> bytes:` ?

`pickle.dumps` function returns bytes type.

Reply:
just want to keep the signature the same as the base class